mautic/mautic
# "Reply To" Address Should Default to the "From Address" In Email Advanced Tab

| Q                                      | A
| -------------------------------------- | ---
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | N.A. 
| Related developer documentation PR URL | N.A. 
| Issue(s) addressed                     | N.A. 

#### Description:
If a "from" email address field is specified in the advanced tab of the email but the "reply to" email address field is blank, the "reply to" will default be the global sender email address specified in Configuration -> Email settings.
If a "from" email address field is specified in advanced tab AND "reply to" is blank, "reply to" should default to the "from" email specified in the advanced tab and not the global default sender email address.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Check default email address in configuration -> email settings
3. Open an email and change the "from" email address in the advanced tab but leave the "reply to" field blank
4. Send test email OR send email using a campaign
5. Reply to the email sent and observe that the "reply to" is not the same as the sender email
6. "Reply to" email should be "from" email address

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10471"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
